### PR TITLE
Added the option to change the title in HopperGUI.java

### PR DESCRIPTION
**Pull request by IoyoCode**
[contributing]: https://github.com/Negative-Games/Framework/blob/main/CONTRIBUTING.md

## Pull Request Etiquette 

 
- [x] I have checked the PRs for upcoming features/bug fixes. 
- [x] I have read the [contributing guidelines][contributing]. 

 
### Changes 
- [ ] Internal code 
- [x] Library interface (affecting end-user code) 
- [ ] Documentation 
- [ ] Other: \_____  

 

Closes Issue: NaN 

## Description 
Exactly the title - going to add the rest of the GUI's soon.

### CODE REVIEW

File: src/main/java/games/negative/framework/gui/HopperGUI.java [hunk 1]
```diff
@@ -213,4 +213,13 @@ public void refresh(@NotNull Player player) {
         });
     }
 
+    /**
+     * Set the title of the GUI
+     *
+     * @param s The new title
+     */
+    public void setTitle(@NotNull String s) {
```
Comment (NegativeDev) [inline src/main/java/games/negative/framework/gui/HopperGUI.java:221]: Do not make the parameter `s`. I recommend making it something like `title` or `input`.

Comment (NegativeDev) [reply src/main/java/games/negative/framework/gui/HopperGUI.java:221]: Secondly, this method is not really required as the class is already annotated with `@Setter` by Lombok, which automatically generates the setter methods for non-final field variables at compile-time

**Review by NegativeDev — CHANGES_REQUESTED** [review]

File: src/main/java/games/negative/framework/gui/HopperGUI.java (hunk 1 above)
Comment (IoyoCode) [reply src/main/java/games/negative/framework/gui/HopperGUI.java:221]: The variable is final, so we can't actually set the title.

Comment (IoyoCode) [reply src/main/java/games/negative/framework/gui/HopperGUI.java:221]: also i do know what `@Setter` does lol

Comment (NegativeDev) [reply src/main/java/games/negative/framework/gui/HopperGUI.java:221]: > The variable is final, so we can't actually set the title.

I know, in your commit you also made the field variable non-final, which is why I mentioned that.

Comment (IoyoCode) [reply src/main/java/games/negative/framework/gui/HopperGUI.java:221]: Ohhh nvm im stupid
ok i'll update this in a few mins, currently working on a commission rn

**Comment by joeecodes** [comment]
Ignore `.github/workflows/issue-project.yml / createCard (pull_request)` checks

**Comment by joeecodes** [comment]
@IoyoCode Please merge the changes in master when pushing a new commit.

**Pull request closed without merging**